### ISSUE TITLE
Pollutes global scope with `Module`

### ISSUE THREAD

**Issue by davidyuk** [antelle/argon2-browser#67]
Running `hash` defines a global variable `Module` with some internal stuff. Probably this comes from `emscripten` but would be better to avoid this, because of:
- possible collision that may happen (also may be fixed by changing `Module` to something more specific)
- to don't expose library internals without reason
- in general, to don't define/change global variables should be a good practice

**Comment by antelle** [comment]
I don't think it will be possible to solve without changing how it works. Not closing for visibility, no action will be taken.

**Comment by antelle** [comment]
Actually, I'm wrong, it's possible to change it: https://stackoverflow.com/questions/30152622/changing-the-default-namespace-module-in-emscripten
